orttraining/orttraining/python/training/ortmodule/_inference_manager.py

```diff
@@ -39,7 +39,12 @@ def execution_session_run_forward(execution_session, onnx_model, device, *inputs
         run_options = C.RunOptions()
 
         # Use IO binding
-        _utils._create_iobinding(io_binding, inputs, onnx_model, device)
+        forward_inputs = []
+        for input in inputs:
+            if not input.is_contiguous():
+                input = input.contiguous()
+            forward_inputs.append(input)
+        _utils._create_iobinding(io_binding, forward_inputs, onnx_model, device)
```

Comment:
Did you profile this PR to see if this loop brings performance impact? list.append tends to be quite expensive